src/hardforks/mod.rs

```diff
@@ -162,6 +162,18 @@ pub trait BscHardforks: EthereumHardforks {
         self.bsc_fork_activation(BscHardfork::HaberFix).active_at_timestamp(timestamp)
     }
 
+    /// Convenience method to check if [`BscHardfork::Cancun`] is firstly active at a given
+    /// timestamp and parent timestamp.
+    fn is_on_cancun_at_timestamp(&self, timestamp: u64, parent_timestamp: u64) -> bool {
```

Comment:
I think this is a bit confusing, should we name this `cancun_transactions_at_timestamp`

because this only checks if parent < cancun and timestamp >= cancun